federatedscope/vertical_fl/dataset/abalone.py

```diff
@@ -54,7 +54,6 @@ class Abalone:
 
     def __init__(self,
                  root,
-                 name,
```

Comment:
Maybe we should also remove the "name" line (i.e., line 39) in "Arguments" of the class.